.github/CONTRIBUTING.md

```diff
@@ -48,10 +48,11 @@ vendor/bin/phpunit --group=e2e --list-tests
 Some end-to-end tests are called "non-standard" as in they have their own script. They can be executed with:
 
 ```shell
-./tests/e2e_tests <infection-executable> [<e2e-test>]
```

Comment:
Documentation improvements extracted to https://github.com/infection/infection/pull/2728